src/views/components/package-file-browser.html

```diff
@@ -65,7 +65,7 @@
 
 					<input type="checkbox" class="switch-inline" id="{{name}}" checked="{{getCollectionIndex(~/collection, ~/type, ~/name, ~/version, '/' + pathS + name) !== -1}}" twoway="false">
 					<label for="{{name}}" on-click="@this.toggleCollectionFile('/' + pathS + name)" as-tooltip="'Add to collection'"></label>
-					<i class="fa fa-clipboard" aria-hidden="true" as-clipboard data-clipboard-text="https://cdn.jsdelivr.net/{{~/type}}/{{~/name}}@{{~/version}}/{{pathS}}{{name}}"></i>
+					<c-copy-dropdown url="https://cdn.jsdelivr.net/{{~/type}}/{{~/name}}@{{~/version}}/{{pathS}}{{name}}" hash="{{hash}}" dropup="{{@index && ../length - @index <= 4}}"></c-copy-dropdown>
```

Comment:
minor thing, but when it's shown above the mouse, the tooltip is still visible and covering it, and then it causes a lot of flickering when hovered over. Maybe the tooltip could have `pointer-events: none;`?

Reply:
👍 the tooltip was also covering part of the text so I moved it to the bottom when `dropup=true`